settings/com.endlessm.settings.gschema.override.in

```diff
@@ -13,7 +13,7 @@ user-switch-enabled=false
 # Force the app menu to be on the application window rather than the shell panel
 # (required due to removal of the app menu from the panel)
 [org.gnome.settings-daemon.plugins.xsettings]
-overrides={"Gtk/ShellShowsAppMenu": <int32 0>}
+overrides={"Gtk/ShellShowsAppMenu": <int32 0>, "Gtk/EntryPasswordHintTimeout": <int32 600>}
```

Comment:
Unrelated to this change, but – do we still need to override `Gtk/ShellShowsAppsMenu`? Didn't the upstream shell change its behaviour to not show app menus?